tests/test_models.py

```diff
@@ -8,23 +8,38 @@
 
 class TestModels(unittest.TestCase):
 
+    standard_course = {'course_code': 'ABC123', 'title': 'Foo Course', 'slug': 'abc-123'}
```

Comment:
Not a fan of putting dicts and lists and things as default values to class properties; this might be better in a constructor.

Reply:
Yes, I agree with you. For now, I don't know all the basic data so used that just to shorten code.